main.py

```diff
@@ -286,6 +319,44 @@ def validateSettings(staticLetters) -> bool:
     return True
 
 
+def normalized(sett: dict) -> list:
+    total = sum(sett[i] for i in sett)
+    return [(sett[i]/total, i) for i in sett]
+
+
+def orderLetters(configs: tuple = BIGRAMS_CONFIGS) -> str:
+
+    # Generate monograph sets for each language
```

Comment:
I think the three comments inside this function were meant to say "monogram"
```diff
- # Generate monograph sets for each language
+ # Generate monogram sets for each language
```

Also, again, please only mention the word "set" if the actual data-type used is a `set()`. The goal of this guideline is to avoid unnecessary confusion.

Reply:
Done in 444e09cfcf29a2dc6c1ad61332d4f2e98f975ae6